core/src/main/kotlin/com/strumenta/starlasu/mapping/ParseTreeToASTTransformer.kt

```diff
@@ -20,29 +21,41 @@ import kotlin.reflect.KClass
 open class ParseTreeToASTTransformer
```

Comment:
Do we expect subclasses of this to use a subclass of TransformationContext? If so, should we make this type parametric w.r.t. TransformationContext?

For example:
```
open class ParseTreeToASTTransformer<TC : ParseTreeToASTTransformer> { ...
```

Reply:
Makes sense, marking as resolved